db/migrations/postgres/000030_create_tokentransfer_table.up.sql

```diff
@@ -0,0 +1,19 @@
+BEGIN;
+CREATE TABLE tokentransfer (
+  seq              SERIAL          PRIMARY KEY,
+  local_id         UUID            NOT NULL,
+  type             VARCHAR(64)     NOT NULL,
+  pool_protocol_id VARCHAR(1024)   NOT NULL,
+  token_index      VARCHAR(1024)   NOT NULL,
+  from_identity    VARCHAR(1024)   NULL,
```

Comment:
Are these really `identity`, or are they actually `key`?

... I know we're broken in `organization` at the moment, where the field is `identity`, but the contents are a `key` - and that needs to be fixed. I'm just concerned about propagating that problem any further.